packages/@aws-cdk/aws-sagemaker-alpha/lib/model-data.ts

```diff
@@ -51,8 +113,33 @@ export abstract class ModelData {
   public abstract bind(scope: Construct, model: IModel): ModelDataConfig;
 }
 
+/**
+ * The options for identifying model artifacts.
+ * When you choose `CompressionType.GZIP` and `S3DataType.S3_OBJECT` then use `ModelDataUrl` property.
+ * Otherwise, use `ModelDataSource` property.
```

Comment:
Reading this, I interpret it as telling the user to use the `ModelDataUrl` or `ModelDataSource` properties, but looking through the rest of the changes, it seems like this is something handled internally. Is there a reason the user needs to know about what's going on under the hood?